app/services/mongodb/MongoDBDatasetService.scala

```diff
@@ -1075,7 +1075,7 @@ class MongoDBDatasetService @Inject() (
   def removeTags(id: UUID, userIdStr: Option[String], eid: Option[String], tags: List[String]) {
```

Comment:
Since we are not using `userIdStr: Option[String], eid: Option[String]` should we remove them from the method definition and refactor the code that calls it?

Reply:
Working on this.